client/src/containers/statisticsPage/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { Progress } from 'antd';
+import axios from 'axios';
+import AdminContainer from '../../components/AdminContainer';
+import './style.css';
+
+class Statistics extends Component {
+  state = {
+    data: {
+      cohortsCount: '',
+      projectsCount: '',
+      studentsCount: '',
+    },
+  };
+
+  async componentDidMount() {
+    try {
+      const {
+        data: { data },
+      } = await axios.get('/api/v1/stats');
+      this.setState({ data });
+    } catch (err) {
+      console.log('There is no data');
```

Comment:
as I mentioned before add notification indicating the error condition instead of console.log